auth_conversion.py

```diff
@@ -0,0 +1,110 @@
+import json
+import sqlite3
+from pathlib import Path
+from uuid import NAMESPACE_URL
+from uuid import uuid5
+
+ASREVIEW_PATH = Path("/Users/casperkaandorp/.asreview")
```

Comment:
Get the asreview path from asreview:

`[python]
from asreview import asreview_path

ASREVIEW_PATH = asreview_path()
`

Reply:
Or better yet, replace the occurences of `ASREVIEW_PATH` by `asreview_path()`

Reply:
I have replaced all occurences of ASREVIEW_PATH with asreview_path